oauth-authorization-server/spring-authorization-server/src/test/java/CustomClaimsConfigurationTest.java

```diff
@@ -0,0 +1,72 @@
+import com.baeldung.OAuth2AuthorizationServerApplication;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.SignedJWT;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import java.text.ParseException;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = OAuth2AuthorizationServerApplication.class)
+@ActiveProfiles(value = "basic-claim")
+public class CustomClaimsConfigurationTest {
+
+    private static final String ISSUER_URL = "http://localhost:";
+    private static final String USERNAME = "articles-client";
+    private static final String PASSWORD = "secret";
+    private static final String GRANT_TYPE = "client_credentials";
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    @LocalServerPort
+    private int serverPort;
+
+    @Test
+    public void givenAccessToken_whenGetCustomClaim_thenSuccess() throws ParseException {
+        String url =  ISSUER_URL + serverPort + "/oauth2/token";
```

Comment:
there is an extra space here between "=  ISSUER_URL"

Reply:
Done